language-support/ts/codegen/tests/ts/build-and-lint-test/src/__tests__/test.ts

```diff
@@ -631,7 +631,10 @@ test("exercise using explicit disclosure", async () => {
     ],
     { encoding: "utf8" },
   );
-  const created = JSON.parse(output).transaction.events[0].created;
+  const transaction = JSON.parse(output).transaction;
```

Comment:
this tests the read side but the thing you're actually modifying is the write side. If you want to test that you probably want to submit two creates with different domain ids and check that hte contracts end on the respective domains. Not sure that's worth the complexity tbh

Reply:
I was thinking we should not go big length here to test the behavior of explicit routing, I just wanted to have a smoke test making sure we are actually passing down the domainId via the JSON-API -> Ledger API tracks. (what this test does).

I guess what is missing here that for an incorrect domainId the JSON-API (or the Ledger API under the hood) yells, if the domain ID is incorrect (this I tested manually, it yelled), but for that I would need some help how to expect errors in a proper way back.